src/test/java/uk/ac/ebi/subs/fileupload/services/globus/GlobusApiClientTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.subs.fileupload.services.globus;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.ConfigFileApplicationContextInitializer;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.ac.ebi.subs.fileupload.config.Config;
+
+@Ignore
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {Config.class, GlobusApiClient.class}, initializers = ConfigFileApplicationContextInitializer.class)
+@TestPropertySource(locations="classpath:application.yml")
+public class GlobusApiClientTest {

```

Comment:
What is the purpose of this class?

Reply:
i created it quickly check if this class initializes and refreshes a token without running the whole application. it does not serve any real purpose but could be useful if we just want to focus on testing if the APIs are functional or not.